content/overview/user-interface/views.md

```diff
@@ -6,14 +6,22 @@ weight: 10
 author: "people"
 ---
 
-The functionality in darktable is separated into six different views:
+The functionality in darktable is delivered through two main views and four subsidiary views:
+
+**Main Views**
```

Comment:
I think this should use markdown "header" syntax (with the `#` symbol) rather than just bold text. Additionally dtdocs has headings in lower case to match the style of the application UI.